sdk/lusid/utilities/api_configuration_loader.py

```diff
@@ -1,55 +1,69 @@
-import os
 import json
-from urllib.request import pathname2url
+import os
+
 from .api_configuration import ApiConfiguration
 
 
 class ApiConfigurationLoader:
 
     @staticmethod
-    def load(api_secrets_filename):
+    def load(api_secrets_filename=None):
         """
         :param api_secrets_filename: name of api configuration file
         :return: populated ApiConfiguration
         """
 
-        # Load our configuration details from the environment variables
-        token_url = os.getenv("FBN_TOKEN_URL", None)
-        api_url = os.getenv("FBN_LUSID_API_URL", None)
-        username = os.getenv("FBN_USERNAME", None)
-        password_raw = os.getenv("FBN_PASSWORD", None)
-        client_id_raw = os.getenv("FBN_CLIENT_ID", None)
-        client_secret_raw = os.getenv("FBN_CLIENT_SECRET", None)
-        app_name = os.getenv("FBN_APP_NAME", "")
+        def vars_from_env():
+            # Load our configuration details from the environment variables
+            return {
+                "token_url": os.getenv("FBN_TOKEN_URL", None),
+                "api_url": os.getenv("FBN_LUSID_API_URL", None),
+                "username": os.getenv("FBN_USERNAME", None),
+                "password": os.getenv("FBN_PASSWORD", None),
+                "client_id": os.getenv("FBN_CLIENT_ID", None),
+                "client_secret": os.getenv("FBN_CLIENT_SECRET", None),
+                "app_name": os.getenv("FBN_APP_NAME", "")
```

Comment:
The environment variable names are hard coded both here and in the dictionary lower down. This leaves room for mismatches in the values.

Reply:
done